lhotse/dataset/unsupervised.py

```diff
@@ -28,11 +30,45 @@ def __getitem__(self, item: int) -> torch.Tensor:
     def __len__(self):
         return len(self.cuts)
 
+    def _validate(self):
+        assert all(cut.has_features for cut in self.cuts)
+
 
 class UnsupervisedWaveformDataset(UnsupervisedDataset):
-    """A special UnsupervisedDataset which provides waveform instead of features."""
+    """
+    A variant of UnsupervisedDataset that provides waveform samples instead of features.
+    The output is a tensor of shape (C, T), with C being the number of channels and T the number of audio samples.
```

Comment:
you really want such long lines?

Reply:
I've always used 120 character limit, but I'm open to change. What's the reason to have the limit shorter and what number are you suggesting?